lib/src/food.ts

```diff
@@ -0,0 +1,205 @@
+import { ThresholdTable } from './rollFromTable'
+
+type CookingMethods =
```

Comment:
I'd run an auto-format in this whole file, to fix any inconsistencies.

Reply:
Ah, my `yarn lint` is still borked- the unnecessary pipe on line 4, right?

Reply:
I'm just talking about formatting, not linting.